core/base/planarGraphLayout/MergeTreeVisu.h

```diff
@@ -0,0 +1,948 @@
+/// \ingroup base
+/// \class MergeTreeVisu
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Visualization module for merge trees.
+///
+
+#ifndef _MERGETREEVISU_H
+#define _MERGETREEVISU_H
+
+#pragma once
+
+#include <FTMTree.h>
+#include <FTMTreeUtils.h>
+
+#include <Debug.h>
+#include <ttkAlgorithm.h>
```

Comment:
to remove.

Reply:
In the next commit: ttkAlgorithm and Debuh will be removed but FTMTreeUtils is still needed.